### ISSUE TITLE
Fatal error: Uncaught Error: Call to undefined function `AMFUnsplash\add_query_args()`

### ISSUE THREAD

**Issue by ntwb** [humanmade/amf-unsplash#3]
I tried updating Altis Local Server using `composer update -W`
- `Upgrading humanmade/amf-wordpress (0.2.2 => 0.2.3)`
- `Upgrading humanmade/asset-manager-framework (0.12.4 => 0.12.5)`

Also confirmed AMF-Unsplash is using v0.2.0 based on changelog here https://github.com/humanmade/amf-unsplash/compare/0.1.1...0.2.0

Reproduction Steps:

- Update Altis Local Server v8 using `composer update -W`
- Download this AMF Unsplash plugin and add to `content/plugins` (Also tried as an `mu-plugin` with same results)

- Test Unsplash by searching for an image in the Unsplash library
- Add an Unsplash image to a post
- Switch the media library back to "Local Media"
- No images appear, the spinner gif constantly spins
- Expected to see the single image already in the local media library and the unsplash image used in the above post

Opening the Console the following isssue is observered:

- `XHR POST | /wp-admin/admin-ajax.php`
- `500 Internal Server Error`
- `Referer | /wp-admin/upload.php`

On the Console Response tab the following stack trace is shown:

Summary: https://github.com/humanmade/amf-unsplash/blob/779c370fd6e936432ca2576a1081851712d224c8/inc/class-provider.php#L358
> `Fatal error: Uncaught Error: Call to undefined function AMFUnsplash\add_query_args()`
> `in /usr/src/app/content/plugins/amf-unsplash-master/inc/class-provider.php on line 358`

Full Stack Trace:
```
Fatal error: Uncaught Error: Call to undefined function AMFUnsplash\add_query_args()
in /usr/src/app/content/mu-plugins/amf-unsplash-master/inc/class-provider.php on line 358

Call stack:

    AMFUnsplash\Provider::resize()
    vendor/humanmade/asset-manager-framework/inc/namespace.php:461
    AssetManagerFramework\dynamic_downsize()
    wp-includes/class-wp-hook.php:292
    WP_Hook::apply_filters()
    wp-includes/plugin.php:212
    apply_filters()
    wp-includes/media.php:4025
    wp_prepare_attachment_for_js()
    array_map()
    wp-admin/includes/ajax-actions.php:2993
    wp_ajax_query_attachments()
    vendor/humanmade/asset-manager-framework/inc/LocalProvider.php:24
    AssetManagerFramework\LocalProvider::request()
    vendor/humanmade/asset-manager-framework/inc/Provider.php:138
    AssetManagerFramework\Provider::request_items()
    vendor/humanmade/asset-manager-framework/inc/namespace.php:273
    AssetManagerFramework\ajax_query_attachments()
    wp-includes/class-wp-hook.php:292
    WP_Hook::apply_filters()
    wp-includes/class-wp-hook.php:316
    WP_Hook::do_action()
    wp-includes/plugin.php:484
    do_action()
    wp-admin/admin-ajax.php:187
```

**Comment by ntwb** [comment]
Fix in #4 resolves the issue and the local media library now loads as expected with AMF Unsplash activated

**Comment by roborourke** [comment]
Whoops. Think I wrote the code for the dynamic image sizing directly in the docs 😬 , thanks for the fix.
